doc/whats_new.rst

```diff
@@ -529,6 +529,62 @@ API changes summary
      (`#6660 <https://github.com/scikit-learn/scikit-learn/pull/6660>`_)
      by `Raghav RV`_.
 
+Contributors
```

Comment:
I'd rather see this as "Code contributors". As someone who does a lot of non-code contributing.